src/containers/CreateBounty/index.js

```diff
@@ -282,8 +282,7 @@ const mapStateToProps = (state, router) => {
     formInitialValues: {
       title: draftBounty.title,
       categories: categories,
-      description:
-        draftBounty.description || intl.get('components.editor.default'),
+      description: intl.get('components.editor.default'),
```

Comment:
@rej156 incase you're curious, this was the line that messed up the bounty description editing